tests/publiccloud/run_ltp.pm

```diff
@@ -47,21 +45,19 @@ sub instance_log_args
 
 sub upload_ltp_logs
 {
-    my ($self) = @_;
     record_info('LTP Logs', 'upload');
-    assert_script_run("test -f $root_dir/result.json || echo No result log");
-    parse_extra_log('LTP', "$root_dir/result.json");
-    # debug file in the standart LTP log-dir. structure:
-    assert_script_run("test -f /tmp/runltp.\$USER/latest/debug.log || echo No debug log");
-    upload_logs("/tmp/runltp.\$USER/latest/debug.log", failok => 1);
+    assert_script_run("test -f /tmp/kirk.\$USER/latest/debug.log || echo No debug log");
+    upload_logs("/tmp/kirk.\$USER/latest/debug.log", failok => 1);
```

Comment:
Let's use a .txt extension as it's more user-friendly. You can open it in a tab instead of downloading it.
```suggestion
    upload_logs("/tmp/kirk.\$USER/latest/debug.txt", failok => 1);
```